packages/open-next/src/build/createAssets.ts

```diff
@@ -44,7 +44,7 @@ export function createStaticAssets(options: buildHelper.BuildOptions) {
 
   const faviconPath = path.join(appPath, appSrcPath, "favicon.ico");
 
-  if (fs.existsSync(faviconPath)) {
+  if (fs.existsSync(faviconPath) && fs.lstatSync(faviconPath).isFile()) {
```

Comment:
+1 for the change.
Maybe add a comment why this is needed.

Reply:
Done!